frame/support/procedural/src/construct_runtime/expand/config.rs

```diff
@@ -97,6 +99,12 @@ pub fn expand_outer_config(
 				Ok(())
 			}
```

Comment:
this code above could be refactored by calling build inside a basic externalities, and we could get rid of `expand_config_build_storage_call` and `BuildModuleGenesisStorage` completely maybe.
But we better to do in a follow-up

Reply:
I planned this in a follow-up.

Reply:
We need to have access `build` function for `RuntimeGenesisConfig` directly, w/o externalities. 

> Code is good to me, maybe it would be great to see the whole picture but this code is good

The broader context is given here: https://forum.polkadot.network/t/genesisconfig-in-wasm-runtime-a-step-towards-a-non-native-future/2887